### PR TITLE
Fixed bug LP: #1607294 snapcraft search returns results in different order

### PR DESCRIPTION
**Pull request by clobrano**
- Accessing search matches' keys in alphabetical order when listing them
- Added test

---

Hello,

I started exploring snapcraft trying to fix some simple bugs. Please let me know if this can be accepted or it needs some changes.

Best regards

### CODE REVIEW

File: snapcraft/tests/test_commands_search.py [hunk 1]
```diff
@@ -21,7 +21,6 @@
 from snapcraft import main, tests
 from snapcraft.tests import fixture_setup
 
-
```
Comment (sergiusens) [inline snapcraft/tests/test_commands_search.py]: why remove this line, I bet static failed here.

File: snapcraft/tests/test_commands_search.py [hunk 2]
```diff
@@ -96,3 +95,19 @@ def test_search_on_non_tty(self):
 curl       test entry for curl
 """
         self.assertEqual(fake_terminal.getvalue(), expected_output)
+
+    def test_search_output_alphabetical_order(self):
+        logger.info('My test')
```
Comment (sergiusens) [inline snapcraft/tests/test_commands_search.py]: Please remove this logging line

**Comment by clobrano** [comment]
Hi,

I made the requested changes. Thank you for the feedback

**Comment by sergiusens** [comment]
ok to test

**Pull request merged**